src/book/03-guides/04-general/04-visualmesh.mdx

```diff
@@ -51,41 +87,40 @@ These lines tell us that the working directory of the container will be the `/wo
 docker build . --pull -t visualmesh:latest
 ```
 
+<details>
+
+<Summary>What does this command do?</Summary>
 - `docker` is the call to docker
 - `build` is the main command being sent to docker
 - `.` is telling docker that the build context is the current directory. Docker will look for a file named `Dockerfile` in the build context telling it what to build (so in this case, the current directory).
 - `--pull` tells docker to always attempt to pull a newer version of the image
 - `-t visualmesh:latest` tells docker that we want to tag the built image as `visualmesh:latest`. We'll see later that when we tell docker to run a container, that we tell docker which image to use by referencing this tag.
 
+</details>
+
 ## Make a dummy dataset
 
-For the sake of this guide, we'll use a dummy dataset containing the same image repeated over and over again. While this will lead to extreme overfitting, it's an easy way to quickly check that the code is running correctly. Download the example image, mask, and lens file [dummy_data.zip](assets/dummy_data.zip), and extract them. Assuming that the file downloaded to `~/Downloads/dummy_data.zip`, then this can be accomplished by the following:
+For the sake of this guide, we'll use a dummy dataset containing the same image repeated over and over again. *While this will lead to extreme overfitting, it's an easy way to quickly check that the code is running correctly.* Download the example image, mask, and lens file [dummy_data.zip](assets/dummy_data.zip) to your downloads folder.
+
+Now, we're going to make a dummy dataset by repeating these files. We can do this using a bash loop. Assuming that we have the dummy data in `~/Downloads/dummy_data`, and we're going to put the repeated data in `~/Downloads/repeated_data`, run the four commands below:
 
 ```bash
 cd ~/Downloads
-unzip dummy_data.zip
-```
 
-### Repeat the data on disk
+unzip dummy_data.zip
 
-Now, we're going to make a dummy dataset by repeating these files. We can do this using a bash loop, such as the following one, which simply prints the numbers 1 to 10 to the terminal:
+mkdir repeated_data
 
-```bash
-for i in {1..10}; do echo ${i}; done
+for i in {1..1000}; do cp ./dummy_data/image.jpg ./repeated_data/image${i}.jpg && cp ./dummy_data/mask.png ./repeated_data/mask${i}.png && cp ./dummy_data/lens.yaml ./repeated_data/lens${i}.yaml; done
```

Comment:
Converted this to a single line for ease of use when copying, as I found the concept to be interpreted in the same way, but with less lines of chunky bash loops